src/Inventory/Asset/Printer.php

```diff
@@ -153,6 +153,31 @@ public function prepare(): array
             }
         }
 
+        //try to know if management port IP is already known as IP port
+        //if yes remove it from management port
+        $port_managment = $this->getManagementPorts();
+        if (isset($this->getManagementPorts()['management']) && property_exists($this->getManagementPorts()['management'], 'ipaddress')) {
```

Comment:
```suggestion
        if (isset($port_managment['management']) && property_exists($port_managment['management'], 'ipaddress')) {
```

Reply:
```$port_managment``` is not used to fetch data, it's used to unset data if need, then is set by ```setManagementPorts```